### PR TITLE
Javadoc

### PR DESCRIPTION
**Pull request by DRSchlaubi**
If relative URLs are broken again please blame @JohnnyJayJay he had 2 months to fix it

### CODE REVIEW

**Comment by DRSchlaubi** [comment]
What teh heck?

Why did you add good first issue to a PR?

And why did you request a review from a guy who has nothing to do with the PR

**Comment by DRSchlaubi** [comment]
Oof

**Comment by jkoenig134** [comment]
Der meinte mich @Seliba

File: src/main/kotlin/com/github/devcordde/devcordbot/commands/general/DocumentedVersion.kt [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    V_1_14_1("SPIGOT", "https://helpch.at/docs/1.14.1/overview-tree.html", "1.14.1"),
+    V_1_14_2("SPIGOT", "https://helpch.at/docs/1.14.2/overview-tree.html", "1.14.2"),
+    V_1_14_3("SPIGOT", "https://helpch.at/docs/1.14.3/overview-tree.html", "1.14.3"),
+    V_1_14_4("SPIGOT", "https://helpch.at/docs/1.14.4/overview-tree.html", "1.14.4"),
+
+    V_1_15("SPIGOT", "https://helpch.at/docs/1.15/overview-tree.html", "1.15"),
+    V_1_15_1("SPIGOT", "https://helpch.at/docs/1.15.1/overview-tree.html", "1.15.1"),
+    V_1_15_2("SPIGOT", "https://helpch.at/docs/1.15.2/overview-tree.html", "1.15.2"),
+
+    V_1_16("SPIGOT", "https://hub.spigotmc.org/javadocs/spigot/overview-tree.html", "1.15.2"),
```
Comment (jkoenig134) [inline src/main/kotlin/com/github/devcordde/devcordbot/commands/general/DocumentedVersion.kt]: 1_16 ->1.15.2

**Review by jkoenig134 — APPROVED** [review]
Looks WONDERFUL to me.

**Pull request merged**